_posts/2019-05-24-announcing-lambda.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Announcing support to run OpenFaaS functions on AWS Lambda
+description: Edward Wilde introduces faas-lambda a way to run OpenFaaS functions on AWS Lambda. Learn how simple container based functions can be deployed to AWS Lambda and Kubernetes with one seamless developer experience
+date: 2019-05-24
+image: /images/openfaas-on-fargate/astronomy-atmosphere-earth-220201.jpg
+categories:
+  - fargate
+  - automation
+  - tutorial
+author_staff_member: ed
+dark_background: true
+---
+
+In this post we are going to examine some of the possible use cases for running OpenFaaS functions on AWS Lambda.
+Then we are going to take a look at some of the new components I've written to make this possible.
+Finally I would like to ask the community for some feedback on these new ideas.
+
+
+## Why would you use OpenFaaS with AWS Lambda?
```

Comment:
`## Why would you use OpenFaaS with AWS Lambda?`

`## What value is there in combining OpenFaaS with AWS Lambda?`

Reply:
change made